cmd/crc/cmd/root.go

```diff
@@ -216,7 +216,7 @@ func attachMiddleware(names []string, cmd *cobra.Command) {
 func defaultTransport() *http.Transport {
 	transport := http.DefaultTransport.(*http.Transport)
 	proxyConfig, err := network.NewProxyConfig()
-	if err != nil {
+	if err != nil || !proxyConfig.IsEnabled() {
```

Comment:
Can you add a `proxy:` prefix to the shortlog? And maybe `if proxy is disabled` instead of `in case proxy is not enable` to keep it short.
 For what it's worth, I'll rework this code some more in a follow-up PR.